static/app/views/performance/traceDetails/traceView.tsx

```diff
@@ -177,6 +191,8 @@ export default function TraceView({
               ...transaction,
               generation,
             }}
+            measurements={getTraceMeasurements(traces[0], generateBounds(traceInfo))}
```

Comment:
The first event in the trace will be the root, this is also to simplify things a bit since there can be multiple roots with measurements, so grabbing the first one instead of having many overlapping measurements seems like the better UI